src/Controls/tests/DeviceTests/Elements/TabbedPage/TabbedPageTests.Android.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Threading.Tasks;
+using AndroidX.ViewPager2.Widget;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.DeviceTests.Stubs;
+using Microsoft.Maui.Graphics;
+using Xunit;
+
+namespace Microsoft.Maui.DeviceTests
+{
+	public partial class TabbedPageTests
+	{
+		[Fact(DisplayName = "Using SelectedTab Color doesnt crash")]
+		public async Task SelectedTabColorNoDoesntCrash()
+		{
+			var expected = Colors.Red;
```

Comment:
Only used to be set in the SelectedTabColor, maybe remove it to make the test simpler?